plugins/accordion/views_bootstrap_accordion_plugin_style.inc

```diff
@@ -14,6 +14,7 @@ class views_bootstrap_accordion_plugin_style extends views_plugin_style {
   public function option_definition() {
     $options = parent::option_definition();
     $options['title_field'] = array('default' => NULL);
+    $options['title_tag'] = array('default' => 'h3');
```

Comment:
I think the current default is H4? So maybe we should change this to H4 instead, to not create any breaking changes for sites that update?

Reply:
fixed